src/stores/PreferenceStore/PreferenceStore.ts

```diff
@@ -255,6 +258,10 @@ const DEFAULTS = {
         pvPreviewCubeSizeLimit: 1,
         pvPreviewCubeSizeLimitUnit: MemoryUnit.GB
     },
+    PV: {
```

Comment:
We usually add to `SILENT` for preferences not listed in the dialog and reset them with `resetSilentSettings`.

Reply:
Thanks for letting me know this 😄. The code is modified accordingly.